test/browser/es6-module-exports-spec.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import TestUtils from 'react-addons-test-utils';
 
-const Divider = require('divider');
+const Divider = require('divider').default;
```

Comment:
I kind of think the same. But that's good to know that `default` is working :grin:. We can rewrite this test when we change the file structure :+1:.